[2024-03-21 16:33:10,446] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: airflow_exam.compute_rf_score 2024-03-21T16:32:00+00:00 [queued]>
[2024-03-21 16:33:10,519] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: airflow_exam.compute_rf_score 2024-03-21T16:32:00+00:00 [queued]>
[2024-03-21 16:33:10,519] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2024-03-21 16:33:10,519] {taskinstance.py:1088} INFO - Starting attempt 1 of 1
[2024-03-21 16:33:10,519] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2024-03-21 16:33:10,548] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): compute_rf_score> on 2024-03-21T16:32:00+00:00
[2024-03-21 16:33:10,563] {standard_task_runner.py:52} INFO - Started process 11292 to run task
[2024-03-21 16:33:10,597] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', '***_exam', 'compute_rf_score', '2024-03-21T16:32:00+00:00', '--job-id', '9419', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/DAG.py', '--cfg-path', '/tmp/tmpe7x8u2ye', '--error-file', '/tmp/tmpma172wp4']
[2024-03-21 16:33:10,598] {standard_task_runner.py:77} INFO - Job 9419: Subtask compute_rf_score
[2024-03-21 16:33:10,842] {logging_mixin.py:104} INFO - Running <TaskInstance: ***_exam.compute_rf_score 2024-03-21T16:32:00+00:00 [running]> on host 3a3d0401ac26
[2024-03-21 16:33:11,001] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=***_exam
AIRFLOW_CTX_TASK_ID=compute_rf_score
AIRFLOW_CTX_EXECUTION_DATE=2024-03-21T16:32:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-03-21T16:32:00+00:00
[2024-03-21 16:33:15,826] {python.py:151} INFO - Done. Returned value was: None
[2024-03-21 16:33:15,841] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=***_exam, task_id=compute_rf_score, execution_date=20240321T163200, start_date=20240321T163310, end_date=20240321T163315
[2024-03-21 16:33:15,885] {taskinstance.py:1265} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-03-21 16:33:15,911] {local_task_job.py:151} INFO - Task exited with return code 0
